packages/react/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@axa-fr/react-oidc",
   "version": "6.15.8",
   "private": false,
+  "type":"module",
```

Comment:
Thank you @jafin for the Awesome PR.

When i run npm install, I have the error bellow.

![image](https://user-images.githubusercontent.com/52236059/225286455-4d75fc9e-6c58-496c-a5b5-0f998e831c42.png)